style.css

```diff
@@ -94,3 +112,180 @@ section .social-media-links {
 section .icon-right {
   margin-right: 20px;
 }
+
+.works {
+  background-image: none;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-auto-rows: minmax(150px, auto);
+}
+
+.works section {
+  margin-bottom: 70px;
+  border-radius: 20px;
+  border: 0.2rem solid #e5e5e5;
+  border-color: #e5e5e531;
+  padding: 16px 30px 25px 16px;
```

Comment:
- [ ] Please add the correct padding-right value to match the given design.
[Hint]: you can make it like that `padding: 16px;` 
- [ ] and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/170547107-c219e39d-81cc-4209-9794-1778860636fe.png)  | ![image](https://user-images.githubusercontent.com/68796232/170547091-337661b5-7de9-451f-809a-e3db849049d1.png)  |